lib/ingram_micro/elements/return_authorization_submission.rb

```diff
@@ -1,67 +1,69 @@
-class IngramMicro::ReturnAuthorizationSubmission < IngramMicro::BaseElement
+module IngramMicro
+  class IngramMicro::ReturnAuthorizationSubmission < IngramMicro::BaseElement
+    DEFAULTS = {
+      customer: nil,
+      shipment_information: nil,
+      credit_card_information: nil,
+      order_header: nil,
+      detail: nil,
+      purchase_order_information: nil,
+      customer_id: nil,
+      business_name: nil,
+      carrier_name: nil
+    }.freeze
 
-  DEFAULTS = {
-    customer: nil,
-    shipment_information: nil,
-    credit_card_information: nil,
-    order_header: nil,
-    detail: nil,
-    purchase_order_information: nil,
-    customer_id: nil,
-    business_name: nil,
-    carrier_name: nil
-  }
-
-  def defaults
-    DEFAULTS
-  end
+    def defaults
```

Comment:
is there a reason you're redefining this in each class? if you only put it in base class and sub class it, does it look up the right `DEFAULT` ?